### PR TITLE
Tutorial 6: Off-Chain Storage

### LINKED ISSUE #301: Run through Tutorial 6: Off-Chain Storage
Fresh eyes happen only once.
Test the docs, note opportunities and friction, and develop user empathy.
This issue scope includes [Tutorial 6: Off-Chain Storage](https://docs.minaprotocol.com/zkapps/tutorials/offchain-storage)

This issue output can then be evaluated for possible improvements if any.

### PR DESCRIPTION
**Pull request by barriebyron**
Based on Gar's advice, remove line numbers to reduce maintenance since the code  no longer aligns with the doc code examples and descriptions now that https://github.com/o1-labs/docs2/pull/568 is merged

This PR includes overall improvements to Tutorial 6: Off-Chain Storage with particular sensitivity to how we present our messaging.

> This tutorial provides a single-server solution to data storage for prototyping zkApps and building zkApps where some trust guarantees are reasonable.
> 
> The solution proposed in this learning tutorial is appropriate for development, but is not recommended for zkApps that require trustlessness. The single-server solution for prototyping is intended as one of several options for data availability on Mina.

- To improve SEO, replace gerunds with active verbs to match search patterns "Implement a Project.." instead of "Implementing a Project..."
- Follow and adhere to tutorial standards in https://github.com/o1-labs/docs2/wiki/Tutorial-standards
- Adhere to docs style guidance https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide

Closes #301 
Preview https://docs2-git-tuto6-update-minadocs.vercel.app/zkapps/tutorials/offchain-storage

### CODE REVIEW

**Comment by barriebyron** [comment]
Needs testing after https://github.com/o1-labs/docs2/pull/568/ is merged

File: docs/zkapps/tutorials/06-offchain-storage.mdx [hunk 1]
```diff
@@ -176,63 +205,74 @@ Start by adding our imports:
  10   PublicKey,
  11   Permissions,
  12   Bool,
- 13 } from 'snarkyjs';
+ 13 } from 'o1js';
  14
  15 import {
  16   OffChainStorage,
  17   Update,
```
Comment (barriebyron) [inline docs/zkapps/tutorials/06-offchain-storage.mdx:212]: No more Update function? let's talk

File: docs/zkapps/tutorials/06-offchain-storage.mdx [hunk 2]
```diff
@@ -282,39 +321,51 @@ Continuing, here is our update function:
...
 
-Note that it contains logic for both running the contract locally and for deploying and interacting with it on Berkeley. This is a useful pattern when developing a new contract. If you would like to try it on Berkeley, deploy our contract as usual with `zk deploy`, set `useLocal` to false in `main.ts`, and add the name of your config to the end of your call to `node main.js`.
+The `main.ts` file contains logic for running the contract locally and for deploying and interacting with it on Berkeley. 
 
-To start, we will connect to the off-chain storage server:
+This is a useful pattern when developing a new contract. 
+
+### Connect to the off-chain storage server
+
+To try your contract on Berkeley, deploy the contract as usual with `zk deploy`, set `useLocal` to false in `main.ts`, and add the name of your config to the end of your call to `node main.js`.
```
Comment (barriebyron) [inline docs/zkapps/tutorials/06-offchain-storage.mdx:350]: @Trivo25  can you help here? what are the specific steps that we are asking our reader to do here? let's provide guided steps and examples

**Comment by barriebyron** [comment]
finish tutorial by removing line numbers to reduce maintenance after code updates

**Review by Trivo25 — APPROVED** [review]

**Review by ymekuria — APPROVED** [review]
Great updates!

**Pull request merged**